robot_skills/src/robot_skills/ebutton.py

```diff
@@ -7,16 +7,21 @@
 
 
 class EButton(RobotPart):
-    """
-    Interface to amigo emergency switch. Listens to /emergency_switch topic
-    """
+    def __init__(self, robot_name, tf_listener, topic=None):
+        """
+        Interface to amigo emergency switch. Listens to /emergency_switch topic
```

Comment:
I think the behaviour of HERO is inversed compared to AMIGO and SERGIO, so this should be checked and incorperated.

Reply:
Nope, same behaviour